misc/report_template.md

```diff
@@ -1,3 +1,5 @@
+# Device {{ clean_mac }}, {{ start_time }} to {{ end_time }}
```

Comment:
Thanks for having a look at this.
The original request in [faucetsdn#134](https://github.com/faucetsdn/daq/issues/134) is to include device_info.make and device_info.model in the report title. Is this going to be addressed in a different PR?

Reply:
This PR was just for the framework changes (making the title as part of the template) -- I was going to leave it up to somebody else to work through the exact formatting and content. Now that it's part of the template it can really can be whatever somebody wants.

Reply:
Ah, makes perfect sense!
I keep on asking for the fish rather than picking up the fishing rod! ;-)
I'll do a PR adding the device_info stuff once this is merged. Thank you!